package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/central-ledger",
-  "version": "9.4.0",
+  "version": "9.4.1",
```

Comment:
Please revert to 9.4.0 as we have not made a release as of yet.

Reply:
ok